### ISSUE TITLE
Interface to PDBRecords

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#12]
**Reported by oliver on 25 Apr 38772760 22:40 UTC**
It would be very much desirable to have a generic user interface to PDBRecords from BALLView.
PDBRecords could be attached to the systems and thus allow the (more or less) lossless reading and writing of PDB files.

**Comment by anhi** [comment]
**Commented by akdehof on 15 Feb 40936094 19:21 UTC**
Deferred to 1.4.1

**Comment by anhi** [comment]
**Commented by wolfgang on 5 Nov 41140893 16:57 UTC**
I've created a patch to carry over PDBInfo (and with that, PDBRecords) into the BALL::Structure that is read from a PDBFile. To completely fulfill this bug, an interface in VIEW would still be needed.

However, the patch creates a regression in XDRPersistenceManager which I just cannot seem to track down, mainly due to not being familiar with the exact inner workings of the Persistence framework. It looks like a data block is being executed when trying to read back the System. I'm probably just missing a type registration somewhere, but I will have to leave this to someone more knowledegeable. 

Here is the relevant output of GDB for such a run:
```
$ gdb bin/TEST/XDRPersistenceManager_test
(gdb) run -v
checking [EXTRA] full_test2... 
Program received signal EXC_BAD_INSTRUCTION, Illegal instruction/operand.
0x000000010076ac4e in typeinfo name for BALL::Object ()
(gdb) bt
#0  0x000000010076ac4e in typeinfo name for BALL::Object ()
#1  0x000000010007c8be in BALL::PersistenceManager::readObject (this=0x7fff5fbf9040) at /Users/wolfgang/Documents/Projects/BALL/source/CONCEPT/persistenceManager.C:310
#2  0x000000010001a7c7 in main (argc=2, argv=0x7fff5fbff578) at /Users/wolfgang/Documents/Projects/BALL/source/TEST/XDRPersistenceManager_test.C:525
(gdb)  
```

**Comment by anhi** [comment]
**Commented by dstoeckel on 5 Apr 41343529 16:37 UTC**
Remove Milestone. Wolfgang should set it to the proper version upon commit.

**Comment by anhi** [comment]
**Commented by wolfgang on 19 Dec 41414255 19:03 UTC**
Silly me, I forgot a ```BALL_CREATE()```

Library support has already made it into master as [plus supporting commits.

I just commited the VIEW part as [50d8e1d1f70e9f3f257f61c6b731bfbd7cff4059](6b2a86c46eda3b016684e2f500c39cb9f5c31e45]).

I hereby propose this as fixed for 1.4.1 ;)
